### DISCUSSION TITLE
Output format options (e.g., html or pdf) in `tar_quarto`

### DISCUSSION THREAD

**Discussion by mattocci27** [ropensci/tarchetypes#102] [Help]
When we have multiple format options in a qmd file as following, how can we choose a format using tar_quarto? 

```{r}
# Unparameterized Quarto document:
lines <- c(
  "---",
  "title: report.qmd source file",
  "format:",
  "  html:",
  "    toc: true",
  "    html-math-method: katex",
  "    css: styles.css",
  "  docx:",
  "    toc: true",
  "    number-sections: true",
  "    highlight-style: github",
  "---",
  "Assume these lines are in report.qmd.",
  "```{r}",
  "targets::tar_read(data)",
  "```"
)
writeLines(lines, "report.qmd")

# Include the report in a pipeline as follows.
targets::tar_script({
  library(tarchetypes)
  list(
    tar_target(data, data.frame(x = seq_len(26), y = letters)),
    tar_quarto(report, path = "report.qmd")
  )
}, ask = FALSE)
# Then, run the pipeline as usual.
```

tar_render accepts an argument for output_format (e.g., tar_render(…, output_format = “word_document”) but tar_quarto doesn't seem to accept that kind of arguments. Thanks!

**Comment by wlandau** [comment]
`tarchetypes` needs to know in advance exactly which files Quarto will generate, and it finds that out with `quarto::quarto_inspect()`. `quarto_inspect()` does not have an `output_format`, so Quarto cannot tell us which files it will generate, and this is not something `tar_quarto()` can reliably guess. See the comments at https://github.com/quarto-dev/quarto-r/issues/14.
